src/Panel/Panel.less

```diff
@@ -0,0 +1,45 @@
+@import (reference) "../less/index";
+
+.Panel--container {
```

Comment:
Can you elaborate on what you mean here? I see
`.TabBar--Tab--alert` in that page and thought container would be like alert?